ui/src/app/app-router.tsx

```diff
@@ -79,12 +79,12 @@ export const AppRouter = ({popupManager, history, notificationsManager}: {popupM
                         navItems={[
                             {
                                 title: 'Workflows',
-                                path: workflowsUrl + '/' + namespace,
+                                path: workflowsUrl + (Utils.managedNamespace ? '' : '/' + namespace),
```

Comment:
I think you can simplify this to `Utils.getNamespace(namespace)`?

Reply:
So, this is actually blanking out the namespace entirely. The goal is to just have it point to /workflows rather than /workflows/namespace. I think, ideally, in managed namespace mode, you don't even need to be aware of the NS. This is how it went in V2, as well.

What say you?

Reply:
I think we should keep the long URLs, they should work for deep-linking:

`/workflows/not-found`

Should result in an error if your namespace is not found.

But perhaps in managed namespace mode:

`/workflows`

Should only load from the managed namespace, rather that attempt/fail to redirect

Reply:
Yeah, that's what my PR does. It overrides the state within each page to point to the managed namespace, so that browsing to

`/workflows`

Only loads from the managed namespace. Thus why I'm having the navbar point just to the root URL for each page, again only in managed mode.

With these changes, when in managed mode, an attempt to browse to something like `/workflows/other_namespace` will redirect you back to `/workflows`. 

In unmanaged mode, the same behavior of not-found namespaces remains.